### DISCUSSION TITLE
重新安装lantern后我的账户就变成免费版了，请客服人员帮我找回我的激活码，感谢

### DISCUSSION THREAD

**Discussion by JasperLiz** [getlantern/lantern-forum-cn#350] [Q&A]
如题，重新安装后，我的账户就变成免费版了，请客服人员帮我找回我的激活码。
我的注册邮箱是869583806@qq.com，下面是我购买一年版VIP专业版的订单记录
![6b001693f93af34aeaf761390efb367](https://github.com/getlantern/lantern-forum-cn/assets/165669006/7fa29c99-c8e9-4af7-9ff9-273071bae588)

**Comment by MaxMKMax** [comment]
@JasperLiz 好的. 我帮您对接负责人安排一下. 这两天请留意邮箱或者垃圾邮箱. 谢谢您!

**Comment by MaxMKMax** [comment]
@JasperLiz 您好. 支付部门说已经帮您升级回去专业版了. 您能看到专业版了吗? 如果不能, 尝试用邮箱登入进去. 他们是说已经帮您手动升级好了.

**Reply by JasperLiz** [reply]
还是不行，我这边用的是windows版本，打开网页后显示还是免费版，而且网页版找不到用邮箱登录的入口

**Reply by JasperLiz** [reply]
如图，进入后还是免费版，而且没有登陆入口
![Uploading image.png…]()

**Reply by JasperLiz** [reply]
![Uploading 微信图片_20240403221754.png…]()

**Reply by JasperLiz** [reply]
在 Windows网页版里只能找到这个激活专业版的入口，但需要输入激活码
![Uploading image.png…]()

**Reply by JasperLiz** [reply]
![Uploading 微信图片_20240403222036.png…]()

**Comment by MaxMKMax** [comment]
@JasperLiz 
您好. 客服回复了. 他说您可以尝试通过电子邮件授权设备就可以用了!

**Reply by JasperLiz** [reply]
已解决，谢谢您和您的同事

**Reply by MaxMKMax** [reply]
@JasperLiz 您好. 不客气的. 谢谢您支持蓝灯!

**Comment by JasperLiz** [comment]
您好，我的专业版又变成免费版了，麻烦再帮我处理一下吧











发自我的iPhone

---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月6日(周六) 晚上8:42
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 重新安装lantern后我的账户就变成免费版了，请客服人员帮我找回我的激活码，感谢 (Discussion #350)




 
@JasperLiz 您好. 不客气的. 谢谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@JasperLiz 好的, 我再帮您提交问题多一次. 不好意思啊.
